### PR TITLE
Add Faction Endings to Hub 01

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Content "Add faction endings to Hub 01"

#### Purpose of change
Every other major faction currently implemented has a faction ending, except the Robofac, or Hub 01. This change will rectify that.

#### Describe the solution
Add thematically appropriate endings for Hub 01.

#### Describe alternatives you've considered
N/A

#### Testing
Put in the game. Did mission. Confirm it shows up.

#### Additional context
There is no way to even get a faction's power to 150, except for the Commune. I'm using @mlangsdorf 's sample epilogues posted in the CDDA development discord.

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
@mlangsdorf just said on the discord:

> My epilogues don't mention them [the Hub 01 maintenance staff] because I tossed them off in 5 minutes as a sample, not as a long considered and reviewed submission.

If anyone can give me improvements i'll be happy to add them!

File: data/json/snippets/epilogue_factions.json [hunk 1]
```diff
@@ -47,6 +47,14 @@
       {
         "id": "epilogue_faction_hells_raiders_150",
         "text": "    Fueled by drugs and rage, the Hell's Raiders fought tooth and nail to overthrow the last strongholds of the Old Guard.  The costly victories brought the warlords abundant territory and slaves but little in the way of stability.  Within weeks, infighting led to civil war as tribes vied for leadership of the faction.  When only one warlord finally secured control, there was nothing left to fight for… just endless cities full of the dead."
+      },
+      {
+        "id": "epilogue_faction_robofac_0",
+        "text": "    Despite Melchior's enlightened leadership, shortages and crisises plagued Hub-01 from the first day of the Cataclysm.  The researchers and administrators among the surviving staff lacked the practical skills to survive in the savage new world.  As Hub-01's systems collapsed from a lack of maintenance, the few survivors fled the building to be killed by the zombies and bandits."
```
Comment (Rivet-the-Zombie) [inline data/json/snippets/epilogue_factions.json:53]: crisises -> crises

File: data/json/snippets/epilogue_factions.json [hunk 2]
```diff
@@ -47,6 +47,14 @@
...
+      },
+      {
+        "id": "epilogue_faction_robofac_150",
+        "text": "       Melchior's enlightened leadership, combined with practical skills provided by mercenary survivors, allowed Hub-01 to flourish briefly.  It became a mecca for advanced technology, selling off devices and equipment that could no longer be reproduced.  When the mercenaries died, Hub-01 fell into disrepair and ruin.  After the air processing facility was overtaken by mold, the remaining staff of Hub-01 fled the building and were killed by zombies and bandits.  Melchior's drones were incapable of maintaining themselves without human hands and stopped working a few months later.  Without regular maintenance, Melchior's own systems also fell to ruin, and less than a decade after the Cataclysm, humanity's first AI powered down and was forgotten."
```
Comment (actual-nh) [inline data/json/snippets/epilogue_factions.json:57]: How did the mercenaries die? Was the problem a lack of teaching of successors, lack of recruitment of successors, or what?

Comment (Xenomorph-III) [reply data/json/snippets/epilogue_factions.json:57]: Likely recruitment, since they'd surely train any children they had!

Comment (actual-nh) [reply data/json/snippets/epilogue_factions.json:57]:
```suggestion
        "text": "       Melchior's enlightened leadership, combined with practical skills provided by mercenary survivors, allowed Hub-01 to flourish briefly.  It became a mecca for advanced technology, selling off devices and equipment that could no longer be reproduced.  Despite this, Hub-01 was never able to expand sufficiently to renew its own population, nor was it able to unite with any other prosperous community, nor could enough trustworthy mercenaries (or staff) be recruited to replace those who died in its defense.  Hub-01 fell into disrepair and ruin.  After the air processing facility was overtaken by mold, the remaining staff of Hub-01 fled the building and were killed by zombies and bandits.  Melchior's drones were incapable of maintaining themselves without human hands and stopped working a few months later.  Without regular maintenance, Melchior's own systems also fell to ruin, and less than a decade after the Cataclysm, humanity's first AI powered down and was forgotten."
```

**Pull request merged**